interpreter.asm

```diff
@@ -244,103 +250,90 @@ FIND
     jsr R_TO
     jmp ZERO
 
-    +BACKLINK
-    !byte	9
-    !text	"find-name"
+    +BACKLINK "find-name", 9
```

Comment:
The proposal for `FIND-NAME` returns a name token, not an execution token.  With the new dictionary structure, it's useful to get the name token for a bunch of reasons.  Right now, those needs are supplied by `DOWORDS`, but it's a roundabout, unclear, and code and cycle heavy way of doing it when `FIND-NAME` skips this token on the way to `>XT`.

I think it would make sense to change the signature to return a name token per the proposal, then refactor `FIND` to make the `>XT` call.

Reply:
After experimentation, I don't think this refactor is worth it.  `HIDE` and `SHOW` will just be bigger words than needed.  If we need code space this could be revisited.